### PR TITLE
DSERV-321 Merging Ontology API endpoints

### PR DESCRIPTION
**Pull request by pedrohr**

### CODE REVIEW

File: data/schema-config.yaml [hunk 1]
```diff
@@ -352,12 +352,12 @@ ontology term:
...
     text:
       type: inverted
-      fields: term_name
+      fields: term_name, description
   accessible_via:
     name: ontology_terms
-    filter_by: _id, term_id, term_name, source
-    fuzzy_text_search: term_name
-    return: _id, uri, term_id, term_name, description, source
+    filter_by: _id, term_id, term_name, source, subontology
```
Comment (mingjiecn) [inline data/schema-config.yaml:358]: Do you need to also index subontology if subontology is a filter?

File: src/routers/genericRouters/routerFuzzy.ts [hunk 2]
```diff
@@ -19,28 +19,55 @@ export class RouterFuzzy extends RouterFilterBy implements Router {
     return `${this.dbCollectionName}_fuzzy_search_alias`
   }
 
-  async getObjectsByFuzzyTextSearch (term: string, page: number, customFilter: string = ''): Promise<any[]> {
-    // supporting only one search field for now
-    let searchField = this.fuzzyTextSearch[0]
+  levenshtein (searchField: string, searchTerm: string): string {
```
Comment (mingjiecn) [inline src/routers/genericRouters/routerFuzzy.ts:22]: Can you add comment doc for those three methods: levenshtein, multipleToken and autocomplete. Thank youQ

**Review by mingjiecn — CHANGES_REQUESTED** [review]
Can you please add comments for the newly added functions? Thank you!

File: src/routers/genericRouters/routerFuzzy.ts (hunk 2 above)
Comment (pedrohr) [reply src/routers/genericRouters/routerFuzzy.ts:23]: Sure! Added! Let me know if I should extend it

**Review by mingjiecn — APPROVED** [review]

**Pull request merged**